### ISSUE TITLE
PCB Design Files

### ISSUE THREAD

**Issue by michaelmoskie** [cooljjj33/PolyDye#3]
Hey, I know you're releasing the files when it feels right, so I'm not going to push too hard.

I'd like to take a look at the PCB/SCH files if you could share those.

Best,
Mike

**Comment by Shaedil** [comment]
Me too

**Comment by cooljjj33** [comment]
We did not expect PolyDye to become so popular so quickly as we had planned a staggered release of all of the components. Both Jacques and I have full time day jobs and we only work on PolyDye on our off hours. 

The project is open-source but as stated on the github repo it is incomplete and we are working on getting first the gerbers up then followed by the design files and then finally the source code.

**Comment by dragkid** [comment]
Hey,
I love this product and have a few questions. I'm sorry if I seem pushy; please take your time.
Is it possible to get this to work with a Bambu printer, like the p1s?

If the releases are staggered could you give me an estimated time you think the Gerber and Bom would be uploaded, and when the rest of the Github repo would be completed?

Thanks

**Comment by BrainCoder** [comment]
Any news on this?

**Comment by Gogny** [comment]
Hi ! 
Thank you for this great project, it is an impressive achievement that in your off hours.
I am trying to drive ink cartridges for printing on paper (struggling with waveforms). I will be very happy when you will open source the firmware code, I would love to see how you did it !
I think you used the ESP-idf framework, do you think Arduino would have worked ?

Thanks

**Comment by cooljjj33** [comment]
> Hi ! Thank you for this great project, that is an impressive achievement in your off hours. I am trying to drive ink cartridges for printing on paper (struggling with waveforms). I will be very happy when you will open source the firmware code, I would love to see how you did it ! I think you used the ESP-idf framework, do you think Arduino would have worked ?
> 
> Thanks

Hi, thanks, if you are only looking to drive them on paper you can look at the printhead driver project that the PolyDye one is based off so long as it will still be a while before I release the source for PolyDye.  I used ESP-IDF for VSCode as well as Python for Blender and Angular as well as C#.

**Comment by cooljjj33** [comment]
> Any news on this?

I am trying to get to this as soon as I can, there are quite a bunch of firmware fixes I am busy with as well as I am trying to get the Klipper Build out.  Once everything is more stable I will start releasing the files. I just don't want to release files that are not ready yet.

**Comment by JamAndCheese** [comment]
You could leverage the help of the community by releasing the source early. You will still retain oversight and remain the founders, but then we could fill-in for your lack of available hours. We also have other printer models available to experiment.

 Just a thought.

**Comment by stefansjs** [comment]
Yeah we don't need bug-free releases of the software in order to benefit. Just push your code to another GitHub repo. You can do periodic releases on GitHub as you reach certain milestones. We all understand what it's like to have your software not be fully ready; you don't have to worry and it being picture perfect. Unfortunately, it's not really open source until you publish the source.

I'm interested in seeing how the inkjet part of the firmware works. I'm considering how feasible it is to make my own inkjet printer. I just learned that the cartridges do a lot of the heavy lifting, so learning how do drive the cartridges is really interesting to me!
